packages/office-ui-fabric-react/src/components/ProgressIndicator/ProgressIndicator.tsx

```diff
@@ -39,7 +38,7 @@ export class ProgressIndicator extends BaseComponent<IProgressIndicatorProps, {}
       label = title;
     }
 
-    percentComplete = Math.min(100, Math.max(0, percentComplete! * 100));
+    this.props.percentComplete !== undefined ? percentComplete = Math.min(100, Math.max(0, percentComplete! * 100)) : percentComplete = 0;
```

Comment:
This should be written like the following.
`percentComplete = (this.props.percentComplete) ? Math.min(100, Math.max(0, percentComplete! * 100)) : 0;`

Reply:
@battletoilet, The way you wrote this definitely makes more sense, but ultimately I had to change this line and a few other spots so that it would still show the % version if you give 0 for the % and then only show the indeterminate version if it's actually undefined. Let me know if this is still too messy.